### DISCUSSION TITLE
'Start Here' and Dataview

### DISCUSSION THREAD

**Discussion by DenverEllis** [groepl/Obsidian-Zettelkasten-Starter-Kit#7] [Q&A] [answered]
There is a "Start here" file included in the starter kit, but nothing relating to the linked dataview files or kanban. Do you have a guide somewhere for setting this up? The practice of learning dataview might be more difficult for individuals without scripting experience learning this system than for those with it. Naturally, dataview or scripting are not required to setup a Zettlekasten, but they are nice to haves for this starter kit since the file is included.

**Comment by groepl** [comment, marked as answer]
> ... relating to the linked dataview files or kanban

I just added the missing file "Dataview - Kanban Board.md" to the "5_Structure Notes" folder. Keep in mind to install the Dataview plugin for Obsidian.

**Comment by huzun123** [comment]
I have a few questions.
1-Hello, I am a doctor. I create literature notes and permanent notes from the articles I read. Besides these, there are other information notes that I need to collect. Should these notes also be considered as permanent notes?"
2-The tags used in permanent notes should be related to the context, not the note itself, right? In that case, MOCs can be created from these tags

**Reply by clementpoiret** [reply]
Your first question puzzled me too. Now, I see literature notes as "long fleeting note, living until the end of the current book." To me, they are differ from fleeting notes because: 
- They have a long living time,
- For content taking time to be consumed, it seems easier to collect all notes form this same source at the same time.

Once I finish my book, I come back to the literature note and divide its core ideas into multiple permanent notes, then delete the literature note.

I could totally use fleeting notes, but literature notes usually require longer / heavier working sessions to be processed.

I would happily read how you guys are using them!

**Reply by huzun123** [reply]
I have organized my notes into three categories: Literature notes for key takeaways, permanent notes for the most critical insights, and knowledge notes for accumulating relevant information.

**Comment by spartanide** [comment]
This is probably a dumb question, but how do I add the starter kit into obsidian in order to use it? Thank you for any assistance!
